tutorials/scripting/creating_script_templates.rst

```diff
@@ -205,6 +205,8 @@ Base placeholders
 +--------------------------+----------------------------------------------------+
 | ``_CLASS_``              | The name of the new class.                         |
 +--------------------------+----------------------------------------------------+
+| ``_CLASS_SNAKE_CASE_``   | The name of the new class as ``snake_case``.       |
```

Comment:
Please add `(used in GDScript only)`, like there is `(used in C# only)` for `_BINDINGS_NAMESPACE_`. You need to make the table wider or wrap the text to a new line in the right cell.